roles/glance/templates/etc/glance/policy.json

```diff
@@ -16,8 +16,13 @@
     "upload_image": "",
 
     "delete_image_location": "",
+{% if allow_image_locations|default('False')|bool %}
     "get_image_location": "",
     "set_image_location": "",
+{% else %}
+    "get_image_location": "role:admin or role:cloud_admin",
```

Comment:
Why do we want cloud_admin to have this permission when allow_image_locations == false ?